kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -807,26 +196,32 @@ window.ExerciseView = Backbone.View.extend({
         var self = this;
 
         var defaults = {
-            seed: Math.floor(Math.random() * 200),
-            framework: "khan-exercises"
```

Comment:
This settings of 'framework' on the model just made it always khan-exercises, as we should be using a method on the exercise_data_model (get_framework) that reads the data and checks (by reading uses_assessment_items) what the framework is.